tests/Feature/Api/ProcessRequestsTest.php

```diff
@@ -986,4 +986,46 @@ public function testScreenRequested()
         $data = $response->json()['data'];
         $this->assertEmpty($data);
     }
+    
+    /**
+     * Get a list of Requests by Cases.
+     */
+    public function testRequestByCase()
+    {
+        ProcessRequest::query()->delete();
+        $request = ProcessRequest::factory()->create();
+        ProcessRequest::factory()->count(9)->create([
+            'parent_request_id' => $request->id,
+        ]);
+
+        $url = '/requests-by-case?case_number=' . $request->case_number;
```

Comment:
You can define in a constant the URI to use like:
`const API_REQUESTS_BY_CASE = '/requests-by-case';`